packages/react-switch/src/components/Switch/useSwitchStyles.ts

```diff
@@ -1,19 +1,211 @@
 import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
+import { createFocusIndicatorStyleRule } from '@fluentui/react-tabster';
 import type { SwitchState } from './Switch.types';
 
+const rootClassName = 'ms-Switch-root';
+const trackClassName = 'ms-Switch-track';
+const thumbClassName = 'ms-Switch-thumb';
+
 /**
  * Styles for the root slot
  */
-const useStyles = makeStyles({
-  root: theme => ({}),
+const useRootStyles = makeStyles({
+  root: theme => ({
+    '--switch-width': '40px',
+    '--switch-height': '20px',
+    '--switch-thumb-size': '14px',
+
+    position: 'relative',
+    width: 'var(--switch-width)',
+    height: 'var(--switch-height)',
+    display: 'inline-block',
+    userSelect: 'none',
+    touchAction: 'none',
+    verticalAlign: 'bottom',
+  }),
+
+  unchecked: theme => ({
+    ':hover .ms-Switch-thumb': {
```

Comment:
```suggestion
    [`:hover .${thumbClassName}`]: {
```

Reply:
Interestingly, I get a build error specifically when I apply the variable in this way on the root element. I'm unsure why this is.

![image](https://user-images.githubusercontent.com/7192338/132606797-302817b2-a119-47ac-be9e-432071ed7453.png)

Reply:
I played around with it some more, it seems that where `createFocusIndicatorRule` is applied seems to cause it not to work.

Reply:
We'll address this separately after taking a closer look at the build errors.